<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSeleccionarGeneral</class>
 <widget class="QDialog" name="dlgSeleccionarGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtFiltro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QTableView" name="twSeleccionar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btAnadir">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>340</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>AÃ±adir lugar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
